nestor_api/lib/k8s/builders.py

```diff
@@ -92,7 +106,24 @@ def get_sanitized_names(app_config: dict, process_name: str):
     return app, sanitized_process_name, metadata_name
 
 
-def get_variables(app_config: dict):
+def get_secret_variables(app_config) -> dict:
```

Comment:
Should this be inside of `lib.config` ?
(also, don't forget the typing)

Reply:
I would avoid this because we are re-formatting to k8s format here to have a deployment to apply after, so it is not very generic but more k8s oriented getters.